src/components/GameGrid/Rounds/GroupStages.tsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react'
```

Comment:
Might be worth combining this, QuarterFinals, SemiFinals and KnockoutRounds into one component as they are identical.